presto-main/src/main/java/com/facebook/presto/operator/aggregation/SimpleTypedSet.java

```diff
@@ -29,7 +29,7 @@
 import static it.unimi.dsi.fastutil.HashCommon.arraySize;
 
 public class SimpleTypedSet
-        implements TypedSet
+        implements PositionTypedSet
```

Comment:
I just looked at this class hierarchy and this is the only implementation, so lets just remove both the interfaces and rename this class to `TypedSet`. We can add the interfaces back in the future, if we want to have multiple implementations